src/Http/Controllers/CP/LicensingController.php

```diff
@@ -24,6 +25,8 @@ public function refresh(Licenses $licenses)
     {
         $licenses->refresh();
 
+        LicensesSynced::dispatch();
```

Comment:
Move this into the `$licenses->refresh()` method and call the event `LicensesRefreshed`.